lib/ansible/modules/hostname.py

```diff
@@ -558,7 +542,7 @@ def _scrub_hostname(self, name):
         name = to_text(name)
         replace_chars = u'\'"~`!@#$%^&*(){}[]/=?+\\|-_ '
         delete_chars = u".'"
-        table = self._make_translation(replace_chars, u'-' * len(replace_chars), delete_chars)
+        table = str.maketrans(replace_chars, u'-' * len(replace_chars), delete_chars)
```

Comment:
The u-string prefix can also be dropped, while on it:
```suggestion
        table = str.maketrans(replace_chars, '-' * len(replace_chars), delete_chars)
```

Reply:
Done.